editor/components/inserter/item-list.js

```diff
@@ -94,6 +94,7 @@ class ItemList extends Component {
 						>
 							<span className="editor-inserter__item-icon">
 								<BlockIcon icon={ item.icon } />
+								{ item.hasChildBlocks ? <span className="editor-inserter__item-icon-stack"></span> : '' }
```

Comment:
Done!

Reply:
Side-note: If you want to output nothing, prefer `null` instead of an empty string, since the latter actually creates a Node (`Node.TEXT_NODE`) in the DOM.

Demo: https://codepen.io/aduth/pen/mKeyzQ